From 9aeccfdc1257204b18e0c8efa1f18f9a24cdab75 Mon Sep 17 00:00:00 2001
From: Gunnar Beutner <gbeutner@serenityos.org>
Date: Wed, 16 Jun 2021 11:23:34 +0200
Subject: [PATCH 1/2] chdir() to the resource install path at program startup

The game tries to open its resource files using relative paths, and we
install them into /opt, so chdr() there.
---
 src/main.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/main.c b/src/main.c
index 8fa842f..e16c35e 100644
--- a/src/main.c
+++ b/src/main.c
@@ -12,6 +12,8 @@
 #endif
 
 int main(int argc, char** args) {
+  // The game tries to open its resource files using relative paths
+  chdir("/opt/DungeonRush");
   prngSrand(time(NULL));
   // Start up SDL and create window
   if (!init()) {
-- 
2.36.1

